.gitignore

```
@@ -3,5 +3,6 @@ autogen/__pycache__
 torch_np/__pycache__/*
 torch_np/tests/__pycache__/*
 torch_np/tests/numpy_tests/core/__pycache__/*
+torch_np/testing/__pycache__/*
```

```suggestion
**/__pycache__/
```
Let's just ignore them all.

---

Even more, a better syntax may be
```python
# Byte-compiled / optimized / DLL files
__pycache__/
*.py[cod]
```
as per the [oficial gitignore](https://github.com/github/gitignore/blob/main/Python.gitignore).